Gamma_Analysis.py

```diff
@@ -418,12 +418,13 @@ def check_spectra(samples, background, reference):
                             error_spectrum)
 
 
-def main():
-    background = SPEFile.SPEFile("USS_Independence_Background.Spe")
+def main(background_filename="USS_Independence_Background.Spe", reference_filename="UCB018_Soil_Sample010_2.Spe", file_list=None):
```

Comment:
Keep lines under 80 characters.

You can split the arguments across lines, or try shortening arg names (`bg_file`, `ref_file`) and/or define at the top `DEFAULT_BG_FILE = "USS_Independence_Background.Spe"` (although that might not shorten the line much).